scripts/parse-examples

```diff
@@ -22,9 +22,23 @@ parser=./parse-"$lang"
 
 # Run the parser from source to ocaml CST
 #
+# Input files are examples/*.src
+# If the test is supposed to fail, the pattern is examples/*.err.src
```

Comment:
FYI I have a small regression parsing library I use for the pfff parser in Common2.regression_testing.
See an example of use here: https://github.com/returntocorp/pfff/blob/develop/lang_java/parsing/test_parsing_java.ml
with that, you parse a directory of test java files and it stores on the disk in /tmp the current score, and if you modifiy the parser and rerun on the same test directory, it will compare with the old result and display regressions.